### PR TITLE
Add 'lizard' mode and make it default.

### PR DESCRIPTION
**Pull request by arjanverkerk**

### CODE REVIEW

**Comment by lexvand** [comment]
@caspervdw Do you have time to review today?. We want to deliver the code to our customers.

File: src/threedidepth/calculate.py [hunk 1]
```diff
@@ -84,14 +88,21 @@ def _depth_from_water_level(dem, fillvalue, waterlevel):
...
+        """
         try:
             return self.cache[self.LOOKUP_S1]
         except KeyError:
             nodes = self.gr.nodes.subset(SUBSET_2D_OPEN_WATER)
             timeseries = nodes.timeseries(indexes=[self.calculation_step])
             data = timeseries.only("s1", "id").data
             lookup_s1 = np.full((data["id"]).max() + 1, NO_DATA_VALUE)
-            lookup_s1[data["id"]] = data["s1"]
+            lookup_s1[data["id"]] = data["s1"][0]
```
Comment (caspervdw) [inline src/threedidepth/calculate.py:105]: `self.lookup_s1` is also used in the `ConstantLevelCalculator`. Does this break because of the change here?

**Review by caspervdw — COMMENTED** [review]
One question, looks great :) Learned something about the Lizard interpolation too.

File: src/threedidepth/calculate.py (hunk 1 above)
Comment (arjanverkerk) [reply src/threedidepth/calculate.py:105]: No, I noticed that 2D values (1, N) were assigned to a 1D lookup array and made the conversion explicit, that's all. Technically this kind of change should not be part of this pull request, my bad. The time dimension is always one (timestep) long in the context of this library.

**Comment by arjanverkerk** [comment]
Thanks for reviewing! I tried to test not only all the lines, but also all the relevant cases within those (numpy) lines.

**Pull request merged**